app/konnect/org-management/oidc-idp.md

```diff
@@ -46,7 +46,61 @@ authentication.
     * The path does not require a slash (`/`).
 
 1. Click **Save**.
+<!-- copied from okta instructions  as is also relevant to OIDC but needs SME edit for actual details  
+### Map {{site.konnect_short_name}} teams to Okta groups
+
```

Comment:
suggestion: I think we may need one more line break in between. this is what I'm seeing on the preview.
<img width="560" alt="Screenshot 2023-05-02 at 19 19 30" src="https://user-images.githubusercontent.com/7806510/235821760-acb21ec4-a06a-4362-bd24-36b1d40cfd47.png">
```suggestion

<!-- copied from okta instructions  as is also relevant to OIDC but needs SME edit for actual details  
### Map {{site.konnect_short_name}} teams to Okta groups
```